dpnp/backend/backend_iface.hpp

```diff
@@ -467,4 +467,20 @@ INP_DLLEXPORT void mkl_rng_gaussian(void* result, size_t size);
 template <typename _DataType>
 INP_DLLEXPORT void mkl_rng_uniform(void* result, long low, long high, size_t size);
 
+/**
+ * @ingroup BACKEND_API
+ * @brief initializer for basic random number generator.
+ *
+ */
+INP_DLLEXPORT void dpnp_engine_rng_initialize();
+
+/**
+ * @ingroup BACKEND_API
+ * @brief initializer for basic random number generator.
+ *
+ * @param [in]  low    Left bound of array values.
+ *
+ */
+INP_DLLEXPORT void dpnp_engine_rng_initialize(size_t seed);
```

Comment:
I might be better to rename this function into something widely used and recognizable.
For example `dpnp_srand` as it described http://www.cplusplus.com/reference/cstdlib/srand/

Reply:
This is initializer for `engine` in `backend_sycl`, like `dpnp_queue` for queue. Doesn't it confuse backend library user?

Reply:
But on other hand it seems really widely used and recognizable. I will change it